api/TypeDBOptions.java

```diff
@@ -42,6 +42,8 @@
     private Integer sessionIdleTimeoutMillis = null;
     private Integer schemaLockAcquireTimeoutMillis = null;
 
+    private TypeDBOptions() {}
```

Comment:
I found out a user may accidentally construct the object using the constructor. This shouldn't be allowed - users should construct the object using the static factory method `core()` or `cluster()`.